### PR TITLE
ring screen UI

### PR DESCRIPTION
**Pull request by YuliaSimanavets**

### CODE REVIEW

File: RingSizer/Modules/Ring/View/RingViewController.swift [hunk 1]
```diff
@@ -5,24 +5,94 @@
...
+
+        set()
+        setupHeaderStackView()
+        setupConstraints()
     }
 
-    /*
-    // MARK: - Navigation
+//  TODO: - needs to be changed
+    func set() {
```
Comment (vkostechko-applace) [inline RingSizer/Modules/Ring/View/RingViewController.swift:55]: avoid using 'set' and 'get' keywords  in swift, it's "not swift style"

File: RingSizer/Modules/Ring/View/OrangeCustomButton.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+//
+
+import UIKit
+
+final class OrangeCustomButton: UIButton {
+    private let gradient = CAGradientLayer()
+
+    init(colors: [CGColor]) {
+        super.init(frame: .zero)
+        setTitle("Save", for: .normal)
```
Comment (vkostechko-applace) [inline RingSizer/Modules/Ring/View/OrangeCustomButton.swift:15]: 1. please localize 
2. add parameter "title" into init or setup title in place where you'll display it, do not hardcode "save" inside init

File: RingSizer/Modules/Ring/View/RingMeasurementCustomView.swift [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        usSizeTitleLabel.text = data.usSizeTitle
+    }
+
+    private func setupTopStackView() {
+        topTitleStackView.addArrangedSubview(euSizeTitleLabel)
+        topTitleStackView.addArrangedSubview(mmSizeTitleLabel)
+        topTitleStackView.addArrangedSubview(usSizeTitleLabel)
+        topSubtitleStackView.addArrangedSubview(euSizeSubitleLabel)
+        topSubtitleStackView.addArrangedSubview(mmSizeSubitleLabel)
+        topSubtitleStackView.addArrangedSubview(usSizeSubitleLabel)
```
Comment (vkostechko-applace) [inline RingSizer/Modules/Ring/View/RingMeasurementCustomView.swift:92]: you can make UIStackView extension and add syntax sugar function 
BirbankInvest_—_UIStackView_Utils_swift

File: RingSizer/Modules/Ring/View/RingMeasurementCustomView.swift [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+import SnapKit
+import UIKit
+
+struct RingMeasurementCustomViewModel {
+    let euSizeTitle: String
+    let mmSizeTitle: String
+    let usSizeTitle: String
+}
+
+class RingMeasurementCustomView: BaseView {
```
Comment (vkostechko-applace) [inline RingSizer/Modules/Ring/View/RingMeasurementCustomView.swift:17]: final?

File: RingSizer/Modules/Ring/View/RingViewController.swift (hunk 1 above)
Comment (YuliaSimanavets) [reply RingSizer/Modules/Ring/View/RingViewController.swift]: got it, fixed

File: RingSizer/Modules/Ring/View/OrangeCustomButton.swift (hunk 2 above)
Comment (YuliaSimanavets) [reply RingSizer/Modules/Ring/View/OrangeCustomButton.swift]: fixed

File: RingSizer/Modules/Ring/View/RingMeasurementCustomView.swift (hunk 3 above)
Comment (YuliaSimanavets) [reply RingSizer/Modules/Ring/View/RingMeasurementCustomView.swift]: got it and fixed it

File: RingSizer/Modules/Ring/View/RingMeasurementCustomView.swift (hunk 4 above)
Comment (YuliaSimanavets) [reply RingSizer/Modules/Ring/View/RingMeasurementCustomView.swift]: fixed

**Pull request merged**